qa.sh

```diff
@@ -2,4 +2,4 @@
 
 pyflakes app tests tools && \
 black --check app tests tools && \
-python3 -m pytest -s -vv tests/service.py
+python3 -m pytest -s -vv tests/service_postgres.py
```

Comment:
I really, really think we can just accommodate these changes inside the existing `service.py` file, via an environment variable, e.g. `DATABASE_URL=postgresql://user:password@postgres/db`